lib/ansible/modules/cloud/google/gcp_compute_autoscaler_info.py

```diff
@@ -55,8 +55,8 @@
 '''
 
 EXAMPLES = '''
-- name: " a autoscaler facts"
-  gcp_compute_autoscaler_facts:
+- name: get info on a autoscaler
```

Comment:
```suggestion
- name: get info on an autoscaler
```
I guess such details should be left for another PR :)

Reply:
oh dear, I'm adding all kinds of comments like this one -  I say let's fix them while we're looking